lib/Caxy/HtmlDiff/HtmlDiff.php

```diff
@@ -7,14 +7,16 @@
 class HtmlDiff extends AbstractDiff
 {
     protected $wordIndices;
-    protected $oldTables;
-    protected $newTables;
+    protected $newSpecialScript;
+    protected $oldSpecialScript;
```

Comment:
I see you're using "special elements" and "special scripts" somewhat interchangeably -- I think we should be consistent with how we reference this functionality. We have a property in the AbstractDiff class called `$specialCaseTags` which is used for a completely different feature, so this could be very confusing.

I don't have a solid suggestion for the name right off the bat, but perhaps something that indicates isolated diffing? Some examples:
`$tagsToIsolateDiff`
`$tagsToDiffIsolated`
`$tagsToIsolate`
`$isolatedDiffTags`
`$diffIsolationTags`
`$placeholderDiffTags`

I kind of like $tagsToIsolate and $isolatedDiffTags?